apps/contact-service/src/__tests__/routes/clubs/member/getClubContacts.test.ts

```diff
@@ -0,0 +1,425 @@
+import {SqlClient} from '@effect/sql'
+import {generateClubUuid} from '@vexl-next/domain/src/general/clubs'
+import {UriStringE} from '@vexl-next/domain/src/utility/UriString.brand'
+
+import {type ExpoNotificationToken} from '@vexl-next/domain/src/utility/ExpoNotificationToken.brand'
+import {UserNotClubMemberError} from '@vexl-next/rest-api/src/services/contact/contracts'
+import {expectErrorResponse} from '@vexl-next/server-utils/src/tests/expectErrorResponse'
+import {Effect, Option, Schema} from 'effect'
+import {addChallengeForKey} from '../../../utils/addChallengeForKey'
+import {
+  createMockedUser,
+  type MockedUser,
+} from '../../../utils/createMockedUser'
+import {NodeTestingApp} from '../../../utils/NodeTestingApp'
+import {runPromiseInMockedEnvironment} from '../../../utils/runPromiseInMockedEnvironment'
+
+const ADMIN_TOKEN = 'dev'
+const SOME_URL = Schema.decodeSync(UriStringE)('https://some.url')
+let user1: MockedUser
+let user2: MockedUser
+
+describe('Get club contacts', () => {
+  beforeEach(async () => {
+    await runPromiseInMockedEnvironment(
+      Effect.gen(function* (_) {
+        const sql = yield* _(SqlClient.SqlClient)
+
+        user1 = yield* _(createMockedUser('+420733333331'))
+        user2 = yield* _(createMockedUser('+420733333332'))
+
+        yield* _(sql`DELETE FROM club_invitation_link`)
+        yield* _(sql`DELETE FROM club_member`)
+        yield* _(sql`DELETE FROM club`)
+      })
+    )
+  })
+
+  it('Should return club members for users that are in the same club', async () => {
+    await runPromiseInMockedEnvironment(
+      Effect.gen(function* (_) {
+        const app = yield* _(NodeTestingApp)
+        const forClubUuid = generateClubUuid()
+
+        yield* _(
+          app.createClub({
+            body: {
+              club: {
+                clubImageUrl: SOME_URL,
+                name: 'someName',
+                description: Option.some('someDescription'),
+                membersCountLimit: 100,
+                uuid: forClubUuid,
+                validUntil: new Date(),
```

Comment:
This can be a problem. Let's create the club that's valid at least an hour in the future. 

```suggestion
                validUntil: new Date(Date.now() + 1000 * 60 * 60),
```